third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -784,20 +798,31 @@ fn bytecode_dump_enabled(config: &TestConfig, is_last: bool, name: &str) -> bool
 }
 
 /// Checks for diagnostics and adds them to the baseline.
-fn check_diags(baseline: &mut String, env: &GlobalEnv) -> bool {
+/// Returns (has_errors, has_warnings), where has_warnings is ORed with had_warnings,
```

Comment:
ORed?

Reply:
Sorry, the comment was stale.  Should be fixed now.